### DISCUSSION TITLE
QfieldCloud

### DISCUSSION THREAD

**Discussion by bctevad** [opengisch/QField#2515] [Q&A]
Hello friends, i'm using Qfieldcloud.
I can not upload my QGIS project on Qfieldcloud. when i try they send me this message "The entered path contains one QGIS project file. Exactly as it should be."
Please i need help! thank you

**Comment by suricactus** [comment]
Hello, @bctevad ,

Thanks for trying QFieldCloud and reporting this issue.

It is a known issue in QFieldSync-4.0.0-BETA17 that was recently fixed and released in QFieldSync-4.0.0-BETA18. Please give it a try and let us know if it fixes your issue.

Best,
Ivan

**Reply by bctevad** [reply]
Yesss! thank you with QFieldSync-4.0.0-BETA18 the issue is fixed.

**Reply by suricactus** [reply]
Glad to hear that! Thanks for testing our still beta software, let us know if you find something else.

**Reply by bctevad** [reply]
Hello @suricactus , i have an other worry is about Qfield dev on my smartphone android. when i load QGIS project send by cable in Qfield dev on my phone i can not see the edition button to add points or polygons

**Reply by nirvn** [reply]
@bctevad , this is a known issue that has been fixed in preparation for the next beta. You can download a master APK build in the meantime.

**Reply by suricactus** [reply]
@bctevad I guess you are not using the very latest version of QField. I am sure you would love the changes that we had over the weekend ;) .

**Reply by bctevad** [reply]
Thanks to you all, 
i will try this latest version of Qfield
